### PR TITLE
Create cluster-secrets-openstack-OSUOSL

### PR DESCRIPTION
**Pull request by hamzy**
This maps in a set of secrets for the cluster-secrets-openstack-OSUOSL

### CODE REVIEW

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -846,6 +846,24 @@ secret_configs:
...
         namespace: ci
         name: cluster-secrets-openstack-ppc64le
+  # cluster-secrets-openstack-OSUOSL
+  - from:
+      pull-secret:
+        bw_item: quay.io
+        attachment: pull-secret
+      clouds.yaml:
+        bw_item: ppc64le-osuosl-ci
+        attachment: cluster-secrets-osuosl-clouds
```
Comment (stevekuznetsov) [inline core-services/ci-secret-bootstrap/_config.yaml:856]: Does this already exist?

Comment (hamzy) [reply core-services/ci-secret-bootstrap/_config.yaml:856]: Yes.  This was merged https://github.com/openshift/release/pull/12442
and I did
oc create secret generic cluster-secrets-osuosl-clouds --from-file=/home/OpenShift/ppc64le-ci/root/director-ci-test/clouds-osuosl.yaml
Is this what you were looking for?

**Pull request merged**